packages/shared/components/AccountActivityDetails.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+    import { AddressBox, ActivityInclusionStatusPill } from 'shared/components'
+    import { AccountActivity } from '@core/wallet'
+
+    export let activity: AccountActivity
+</script>
+
+<main-content class="flex flex-auto w-full flex-col items-center justify-center space-y-3">
+    <account-status class="flex flex-row w-full space-x-2 justify-center">
+        <ActivityInclusionStatusPill localizationKey={'account.creation'} inclusionState={activity.inclusionState} />
+    </account-status>
+    <AddressBox clearBackground clearPadding isCopyable address={activity.accountId} />
```

Comment:
I don't know if `accountId` is correct. Maybe we should save the `accountAddress` in `AccountActivity` and pass that here